### ISSUE TITLE
Content review of new cards on CST auth landing page

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#62735]

**Comment by laurwill** [comment]
Hi @skylerschain and @mistymg , 

Finally closing the loop on this (very) old ticket! I had some conversations with Skyler about updating "An item needs your attention" to something more clear and specific, like "We need more information from you" — and I think we need a bit more information before we can make that change, is that right Skyler?

We also chatted about this wording with @mattmarino-adhoc , who's currently doing some user testing on My VA that includes a slim alert on claims with "We need more information from you" label. 

I think the next steps here are:
- Skyler and team would need to confirm if "We need more information from you" is accurate in all cases where this alert can appear, and clear it with SMEs
- Matt can let us know about any research findings related to this wording
- Misty can truncate this label for mobile when the time comes — maybe "More information needed" would fit?

Let me know if I missed anything or anyone has updates on this!

**Comment by mistymg** [comment]
Yes, I think a truncated version will work for mobile app. Thanks for thinking of us :)

**Comment by mattmarino-adhoc** [comment]
@mistymg @laurwill @skylerschain The bulk of our testing is now complete, but we are going to recruit more screen reader users since we didn't get to fully test our prototypes with any yet. I'll reach out to you about what I've documented so far, and include you in reviewing findings when the study is complete.

**Comment by laurwill** [comment]
Hi again! @mattmarino-adhoc just pointed out that "More info needed" doesn't communicate that we need stuff from the Veteran (as opposed to stuff VA gathers). So @mistymg when we get to truncating, we'll have to figure out how to say that clearly in the shorter version.

**Comment by mistymg** [comment]
Good call out!

**Comment by laurwill** [comment]
Wow this is ancient! Moving to validate.

As an odd update here, turns out this alert actually doesn't always mean we need something from the Veteran. The exact same alert shows when we request something from the Veteran vs. when we request something from someone else, like another agency or provider. It's not ideal! 

@mistymg @mattmarino-adhoc you may want to track CST changes related to this sad realization, for mobile and My VA purposes:

- [GH ticket](https://www.figma.com/design/OmJF4qXnHcXZ3SJwTNNyzF/Claim-card-alerts?node-id=21-6661&t=7s10RCbfKEcHve4O-4)
- [Figma](https://www.figma.com/design/OmJF4qXnHcXZ3SJwTNNyzF/Claim-card-alerts?node-id=21-6661&node-type=frame&t=qNbGekWFEAoNOKMt-0)
